build.sbt

```diff
@@ -25,4 +25,4 @@ scalacOptions ++= Seq(
 //  "-Xfatal-warnings"
 )
 
-javaOptions in console ++= Seq("-Djna.library.path=/Users/qi/workspace/helm-java-sdk/helm-go-lib")
+//javaOptions in console ++= Seq("-Djna.library.path=/Users/qi/workspace/helm-java-sdk/helm-go-lib")
```

Comment:
Is there any reason not to just remove this line?

Reply:
None, good catch